docs/data-sources/lb_ips.md

```diff
@@ -0,0 +1,46 @@
+---
+page_title: "Scaleway: scaleway_lb_ips"
+description: |-
+Gets information about multiple Load Balancer IPs.
+---
+
+# scaleway_lb_ips
+
+Gets information about multiple Load Balancer IPs.
+
+## Example Usage
+
+```hcl
+# Find IPs by IP address
+data "scaleway_lb_ips" "my_key" {
+  ip_address = "0.0.0.0"
+}
+# Find IPs by IP address and zone
+data "scaleway_lb_ips" "my_key" {
+  ip_address = "0.0.0.0"
+  zone       = "fr-par-2"
+}
+# Find multiple IPs that share the same prefix
+data "scaleway_lb_ips" "my_key" {
+  ip_address = "51.159.*.*"
```

Comment:
Why not having a CIDR notation here? It would make it easier to understand. Considering IP with regexp seems weird.